fabric-structures-v0/src/main/resources/fabric-structures-v0.accesswidener

```diff
@@ -0,0 +1,2 @@
+accessWidener    v1    named
+accessible    field    net/minecraft/world/gen/feature/StructureFeature    STRUCTURE_TO_GENERATION_STEP    Ljava/util/Map;
```

Comment:
Use an accessor mixin than an access widener. access widener causes extra decompilation for source generation.

Reply:
Resolved in a6b4507